pom.xml

```diff
@@ -214,15 +214,15 @@ THE SOFTWARE.
         <version>${slf4jVersion}</version>
       </dependency>
       <dependency>
+        <groupId>org.slf4j</groupId>
+        <artifactId>log4j-over-slf4j</artifactId>
+        <version>${slf4jVersion}</version>
+      </dependency>
+      <dependency>
         <groupId>commons-logging</groupId>
         <artifactId>commons-logging</artifactId>
         <version>1.1.3</version>
       </dependency>
-      <dependency><!-- [JENKINS-4433] -->
-        <groupId>log4j</groupId>
-        <artifactId>log4j</artifactId>
-        <version>1.2.17</version>
```

Comment:
I would recommend that we dependency manage this to 999.DO_NOT_USE (this may need us to add a pom to our local repo).
That way if anyone adds a new dep in the future, the build will fail rather than silently pick up log4j again.